ChildMindInstitute/mindlogger-backend-refactor
# fix: Add new patch to set auto_assign to true on activities and flows tables (M2-7366)

📝 Description

🔗 [Jira Ticket M2-7366](https://mindlogger.atlassian.net/browse/M2-7366)
Changes include:

Created a Python patch to set the auto_assign field to true for existing records in the activities and flows tables.
Registered the patch in cli.py for execution via the CLI.
🪤 Peer Testing

Requires pipenv shell
Requires pipenv sync --dev

Test steps:

Run the patch: Execute the patch using the CLI command python src/cli.py patch exec [M2-7366](https://mindlogger.atlassian.net/browse/M2-7366).

Expected outcome: The auto_assign field in existing activities and flows records is set to true.

Verify database changes: Check the activities and flows tables to ensure that the auto_assign field is updated correctly for both existing and new records.

Expected outcome: All existing records have auto_assign set to true, and the default for new records is set to true.

✏️ Notes
No additional notes.

[M2-7366]: https://mindlogger.atlassian.net/browse/M2-7366?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
